modules/persistent-storage-local-create-cr.adoc

```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage/persistent-storage-local.adoc
+
+[id="local-volume-cr_{context}"]
+= Provision the local volumes
+
+Local volumes can not be created by dynamic provisioning. Instead,
+PersistentVolumes must be created by the local storage provisioner. This
```

Comment:
lets be consistent in using `local storage provisioner` and `local storage operator`. These documents should just use `local storage operator` for the most part.